### ISSUE TITLE
Use with Windows + pyInstaller breaks after 0.81.0 (esp. in CI)

### ISSUE THREAD

**Issue by finger563** [Textualize/textual#5162]
I have a project for which I use textualize to make a very nice TUI (thanks for all the amazing work you all have done!).

I recently upgraded my dependencies from ==0.81.0 to >=0.82.0 to try to update my code to take advantage of the new `is_on_screen` attribute.

However, after updating to 0.82.0 (and even testing 0.83.0), I found that the packaged TUI on Windows does not render at all (and instead spits out ANSI escape sequences to the terminal corresponding to any mouse movement I make within the terminal).

Downgrading the same code back to 0.81.0 and it works just fine in the same system / terminal, so it appears that something about the changes in 0.82.0 and beyond have messed up rendering in this situation (again, only broken on Windows - on MacOS packaged 0.82.0 and 0.83.0 work just fine).

For some additional information, I'm using both `textual` and `textual_plotext` and have set up the following hidden imports in my pyinstaller spec:

```
    'rich',
    'rich.logging',
    'textual',
    'textual.widgets',
    'textual.widgets._tab_pane',
    'textual_plotext',
    'pkg_resources.extern',
    'pkg_resources.py2_warn',
```

I've not seen anything else pointing to new hidden imports we might need to add, and I'm not getting runtime errors or exceptions that would point to such an issue.

**Comment by willmcgugan** [comment]
But it runs on Windows without being packaged?

We did upgrade some dependancies recently. Can't think what might cause this though.

**Comment by finger563** [comment]
@willmcgugan that's correct, if I run it in windows (within a venv) then it works just fine as well.

**Comment by AlikanakelaKarwowski** [comment]
@finger563 did you ever figure out a solution to this? I am having the same issues on v1.0.0

**Comment by finger563** [comment]
> @finger563 did you ever figure out a solution to this? I am having the same issues on v1.0.0

@AlikanakelaKarwowski I did not find a resolution, we are still just using the older version but I've not tried any new releases
